muspinsim_input/muspinsim_input.xml

```diff
@@ -0,0 +1,423 @@
+            <param name="interaction" value="dipolar"/>
+            <param name="di_index" value="1"/>
+            <param name="di_index_2" value="2"/>
+            <param name="di_vector" value="0.9 0.9 0"/>
+            <param name="interaction" value="dipolar"/>
+            <param name="di_index" value="1"/>
+            <param name="di_index_2" value="3"/>
+            <param name="di_vector" value="-0.9 -0.9 0"/>
+            <param name="interaction" value="dissipation"/>
+            <param name="dis_index" value="1"/>
+            <param name="dis_val" value="0.5"/>
+            <param name="average_axes" value="" />
+            <param name="experiment_preset" value="custom" />
+            <param name="field" value="1.5e-2 1.0e-2 1.0e-2"/>
+            <param name="field" value="0.01"/>
+            <param name="polarization_preset" value="custom"/>
+            <param name="polarization" value="1 0 0"/>
+            <param name="time" value="range(0,8.0,1000)"/>
+            <param name="time" value="range(0,1.0)"/>
+            <param name="orientation_preset" value="eulrange"/>
+            <param name="eul_n" value="10"/>
+            <output name="out_file" file="test_4.in" ftype="txt" compare="diff" />
+        </test>
+        <test>
+            <param name="out_file_prefix" value="test_5"/>
+            <param name="spin_preset" value="mu"/>
+            <param name="interaction" value="dissipation"/>
+            <param name="dis_index" value="1"/>
+            <param name="dis_val" value="g"/>
+            <param name="experiment_preset" value="custom" />
+            <param name="field" value="1.0/muon_gyr" />
+            <param name="fitting" value="true"/>
+            <param name="var_name" value="g" />
+            <param name="min_bound" value="0.0" />
+            <output name="out_file" file="test_5.in" ftype="txt" compare="diff" />
+        </test>
+    </tests>
+    <help><![CDATA[
+    Tool to create input parameter file for Muspinsim.
+
+
+    This tool creates a structured text file with keywords and values which describe the system to model for Muspinsim.
+    See muspinsim docs for more information https://muon-spectroscopy-computational-project.github.io/muspinsim/input/.
+
+
+    Muspinsim allows expressions and special functions to be used when defining certain keywords. This tool also allows this.
+    Check the hint at the bottom of each input to see what, if any, special function or expressions can be used.
+
+
+    Default expressions include the use of the operators \+, \-, \*, \/ and \^ for exponentiation.
+
+
+    Default constants include:
+        - pi: ratio of a circle and its diameter
+        - e: base of the natural logarithm
+        - deg: conversion factor between radians and degrees, equivalent to 180/pi
+        - inf: infinity
+
+    Special constants include:
+        - muon_gyr: gyromagnetic ratio of muon (135.5388 MHz/T)
+        - MHz: `1/2*muon_gyr`
```

Comment:
1. This isn't rendered correctly, or at least not as I expected (italics rather than code format, see image)
2. Is it 1/(2*muon_gyr) or (1/2)*muon_gyr?
![](https://user-images.githubusercontent.com/20683271/170507448-f4244467-5a66-4795-b0f9-43b85836541e.png)